### ISSUE TITLE
OverlappingFileLockException resolving dependencies in DefaultGlobalCacheManager.java:212

### ISSUE THREAD

**Issue by alanmyrvold** [gogradle/gogradle#225]
I see exceptions resolving dependencies with com.github.blindpirate.gogradle	version 0.9.

https://gradle.com/s/syde46gozb5qo
https://gradle.com/s/lokpbzxidb66m
https://gradle.com/s/2xkosn3hoi4do
https://gradle.com/s/dwkubkzhfisse
https://gradle.com/s/kupp24rfaa3ws
https://gradle.com/s/hs7whusiam3ns

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':beam-sdks-go-test:resolveBuildDependencies'.
Caused by: com.github.blindpirate.gogradle.core.exceptions.ResolutionStackWrappingException: 
Exception in resolution, message is:
null
Resolution stack is:
+- github.com/apache/beam/sdks/go/test

Caused by: java.nio.channels.OverlappingFileLockException: (No message provided)
at com.github.blindpirate.gogradle.core.cache.DefaultGlobalCacheManager.doGetMetadata(DefaultGlobalCacheManager.java:212)
at com.github.blindpirate.gogradle.core.cache.DefaultGlobalCacheManager.getMetadata(DefaultGlobalCacheManager.java:199)
at com.github.blindpirate.gogradle.core.pack.GlobalCachePackagePathResolver.produce(GlobalCachePackagePathResolver.java:51)
at com.github.blindpirate.gogradle.util.logging.DebugLogMethodInterceptor.invoke(DebugLogMethodInterceptor.java:38)
at com.github.blindpirate.gogradle.core.pack.GlobalCachePackagePathResolver.produce(GlobalCachePackagePathResolver.java:35)
at com.github.blindpirate.gogradle.util.FactoryUtil.lambda$produce$0(FactoryUtil.java:29)
at com.github.blindpirate.gogradle.util.FactoryUtil.produce(FactoryUtil.java:32)
at com.github.blindpirate.gogradle.core.pack.DefaultPackagePathResolver.produce(DefaultPackagePathResolver.java:60)
at com.github.blindpirate.gogradle.util.logging.DebugLogMethodInterceptor.invoke(DebugLogMethodInterceptor.java:38)
at com.github.blindpirate.gogradle.core.dependency.parse.DefaultMapNotationParser.determinePackage(DefaultMapNotationParser.java:103)
at com.github.blindpirate.gogradle.core.dependency.parse.DefaultMapNotationParser.parse(DefaultMapNotationParser.java:69)
at com.github.blindpirate.gogradle.util.DependencySetUtils.parseMany(DependencySetUtils.java:30)
```

**Comment by alanmyrvold** [comment]
Fix proposed in https://github.com/gogradle/gogradle/pull/227

**Comment by blindpirate** [comment]
Closed by `0.9.3`. Thanks @alanmyrvold!
